### DISCUSSION TITLE
Drag windows

### DISCUSSION THREAD

**Discussion by Henry-Hiles** [nwg-piotr/nwg-shell#73] [Ideas]
Hello there! Using this project rn, looks sick! Only thing I find annoying, is that I can't drag windows. If you could implement something similar to hyprlands shortcuts, where super+left click+drag moves windows and super+right click+drag resizes windows then I think this shell would be perfect!

**Comment by nwg-piotr** [comment]
Well, that's the window manager property, so you'd need to ask sway developers. But they refuse to implement features that aren't present in i3, so it would be a waste of time. :)

BTW: try dragging windows with the Alt key. It works well for me, in the way specific to tiling WMs, of course.

**Reply by Henry-Hiles** [reply]
Alr, thx.
